src/components/ContactCard/ContactForm/index.jsx

```diff
@@ -111,19 +52,18 @@ const ContactForm = ({ contact, onSubmit, t }) => (
                   key={name}
                   name={name}
                   icon={icon}
-                  label={t(`field.${name}`)}
+                  label={t(`fields.${name}`)}
```

Comment:
Are you sure about the `fields.name`? Should it be `field.name`?

Reply:
Nevermind. I got it!

Reply:
oui suite aux commentaires sur les fields, label et placeholder, j'ai renommé field en fields